codus/codelog
# Changelog release preview

## Objective
 - Add the feature to preview releases
- Solve #29 
## Notes
- For the sake of simplicity, i decided not to add `windows command prompt` support (at least not now)

## How to test
1. Add unreleased changes
2. Run `codelog release x.x.x -p` or `codelog release x.x.x --preview`
3. Check the console output (You can copy or take a screenshot to compare with an actual release)

## Checklist

[CONTRIBUTING]: https://github.com/codus/codelog/blob/master/CONTRIBUTING.md
